diff --git a/templates/components/search_packages.html.twig b/templates/components/search_packages.html.twig
index 3b2d870..f007d21 100644
--- a/templates/components/search_packages.html.twig
+++ b/templates/components/search_packages.html.twig
@@ -10,7 +10,7 @@
         >
     </div>
 
-    <div>
+    <div data-loading="addClass(opacity-25)">
         {% for package in computed.packages %}
             <div class="component-box">
                 <span class="rectangle {{ package.color }}"></span>
