diff --git a/src/Twig/SearchPackagesComponent.php b/src/Twig/SearchPackagesComponent.php
index 8f5e87d..0871890 100644
--- a/src/Twig/SearchPackagesComponent.php
+++ b/src/Twig/SearchPackagesComponent.php
@@ -4,6 +4,7 @@ namespace App\Twig;
 
 use App\Repository\PackageRepository;
 use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
 use Symfony\UX\LiveComponent\DefaultActionTrait;
 
 #[AsLiveComponent('search_packages')]
@@ -11,6 +12,7 @@ class SearchPackagesComponent
 {
     use DefaultActionTrait;
 
+    #[LiveProp(writable: true)]
     public string $query = '';
 
     public function __construct(
diff --git a/templates/components/search_packages.html.twig b/templates/components/search_packages.html.twig
index 98b265d..3b2d870 100644
--- a/templates/components/search_packages.html.twig
+++ b/templates/components/search_packages.html.twig
@@ -3,6 +3,8 @@
         <input
             type="search"
             value="{{ query }}"
+            data-model="query"
+            {{ stimulus_action('live', 'update') }}
             class="form-control"
             placeholder="Search from {{ computed.packages|length }} packages..."
         >
