diff --git a/assets/controllers/markdown_controller.js b/assets/controllers/markdown_controller.js
index 03677cb..ef72923 100644
--- a/assets/controllers/markdown_controller.js
+++ b/assets/controllers/markdown_controller.js
@@ -1,4 +1,9 @@
 import { Controller } from '@hotwired/stimulus';
+import snarkdown from 'snarkdown';
 
 export default class extends Controller {
+    convert(event) {
+        const rawText = event.currentTarget.value;
+        console.log(snarkdown(rawText));
+    }
 }
diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index cfc7e4e..591c8ed 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -9,6 +9,7 @@
                     class="form-control"
                     placeholder="type some **markdown**!"
                     rows="5"
+                    {{ stimulus_action('markdown', 'convert') }}
                 ></textarea>
             </div>
             <div class="col-1 text-center">
