diff --git a/assets/controllers/markdown_controller.js b/assets/controllers/markdown_controller.js
index ef72923..cb7b7f4 100644
--- a/assets/controllers/markdown_controller.js
+++ b/assets/controllers/markdown_controller.js
@@ -2,8 +2,10 @@ import { Controller } from '@hotwired/stimulus';
 import snarkdown from 'snarkdown';
 
 export default class extends Controller {
+    static targets = ['output'];
+
     convert(event) {
         const rawText = event.currentTarget.value;
-        console.log(snarkdown(rawText));
+        this.outputTarget.innerHTML = snarkdown(rawText);
     }
 }
diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index 591c8ed..f74c445 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -19,6 +19,7 @@
             </div>
             <div
                 class="col-3 markdown-result"
+                {{ stimulus_target('markdown', 'output') }}
             >
 
             </div>
