diff --git a/src/Twig/SearchPackagesComponent.php b/src/Twig/SearchPackagesComponent.php
new file mode 100644
index 0000000..42aa64f
--- /dev/null
+++ b/src/Twig/SearchPackagesComponent.php
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Twig;
+
+use App\Repository\PackageRepository;
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+
+#[AsTwigComponent('search_packages')]
+class SearchPackagesComponent
+{
+    public string $query = '';
+
+    public function __construct(
+        private PackageRepository $packageRepository
+    )
+    {
+    }
+
+    public function getPackages(): array
+    {
+        return $this->packageRepository->search($this->query);
+    }
+}
diff --git a/templates/components/search_packages.html.twig b/templates/components/search_packages.html.twig
new file mode 100644
index 0000000..c9f595d
--- /dev/null
+++ b/templates/components/search_packages.html.twig
@@ -0,0 +1,19 @@
+<div {{ attributes }}>
+    <div class="mb-3">
+        <input
+            type="search"
+            value="{{ query }}"
+            class="form-control"
+            placeholder="Search from {{ this.packages|length }} packages..."
+        >
+    </div>
+
+    <div>
+        {% for package in this.packages %}
+            <div class="component-box">
+                <span class="rectangle {{ package.color }}"></span>
+                <span class="name">{{ package.name }}</span>
+            </div>
+        {% endfor %}
+    </div>
+</div>
diff --git a/templates/main/components.html.twig b/templates/main/components.html.twig
index b85669b..a7812ef 100644
--- a/templates/main/components.html.twig
+++ b/templates/main/components.html.twig
@@ -3,6 +3,7 @@
 {% block body %}
     <div class="container">
         <h1>Find a component</h1>
+
+        {{ component('search_packages') }}
     </div>
 {% endblock %}
-
