diff --git a/templates/components/search_packages.html.twig b/templates/components/search_packages.html.twig
index c9f595d..98b265d 100644
--- a/templates/components/search_packages.html.twig
+++ b/templates/components/search_packages.html.twig
@@ -4,12 +4,12 @@
             type="search"
             value="{{ query }}"
             class="form-control"
-            placeholder="Search from {{ this.packages|length }} packages..."
+            placeholder="Search from {{ computed.packages|length }} packages..."
         >
     </div>
 
     <div>
-        {% for package in this.packages %}
+        {% for package in computed.packages %}
             <div class="component-box">
                 <span class="rectangle {{ package.color }}"></span>
                 <span class="name">{{ package.name }}</span>
